tajo-docs/src/main/sphinx/table_management/tablespaces.rst

```diff
@@ -1,9 +1,26 @@
-*************************************
+***********
 Tablespaces
-*************************************
+***********
 
-Tablespaces in Tajo allow users to define locations in the storage system where the files or data objects representing database objects can be stored. Once defined, a tablespace can be referred to by name when creating a database or a table. Especially, it is very useful when a Tajo cluster instance should use heterogeneous storage systems such as HDFS, MySQL, and Oracle.
+Tablespaces in Tajo allow users to define locations in the storage system where the files or data objects representing database objects can be stored.
+Once defined, a tablespace can be referred to by name when creating a database or a table.
+Especially, it is very useful when a Tajo cluster instance should use heterogeneous storage systems such as HDFS, MySQL, and Oracle.
 
+============================================
+External Table, Managed Table and Tablespace
+============================================
+
+Tajo has two types of table. One is external table. It needs **location** property when the table is created. Using this property, you can create an external table indicating existing external data source.
+For example, if there is already your data as Text/JSON files or HBase table, you can register it as tajo external table.
+Other one is managed table, which means internal table, that is created in Tajo internal table space. You can convert external data to the form that you want and save it in managed table.
```

Comment:
The definition of managed tables is the tables which are created and managed by Tajo. One of their characteristics is that they are stored in directories under Tajo's root directory.